plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/thrift/ThriftHttpMetastoreConfig.java

```diff
@@ -111,4 +113,17 @@ public ThriftHttpMetastoreConfig setAdditionalHeaders(String httpHeaders)
         }
         return this;
     }
+
```

Comment:
I guess this can just live in StaticMetastoreConfig instead of duplicating in ThriftHttpMetastoreConfig and ThriftMetastoreConfig

Reply:
@anusudarsan its a good suggestion.
Initially, I thought about that, for that we need to`ThriftMetastoreClientFactory.create()` to pass the `catalogName` to Default and HTTP factories, is that what you are suggesting as well?

Reply:
not the ThriftMetastoreClientFactory.create(), but as you have now but change ClientFactory constructors to inject StaticMetastoreConfig

Reply:
took a stab on it in the last commit, please check. thanks